README.md

```diff
@@ -40,6 +40,11 @@ P.S. Where all postings before March 2018 have gone to?
 
 ---
 
+## Hi Kottans ))
+I'm Nataliya and these are my first steps in FrontEnd. I realy hope that everyone of us will get his (or her :) ) dream )
+[![SiameseCat](./assets/images/old_fashioned_siamese_cat_with_amazing_eyes.jpg)](https://github.com/kottans/frontend)
```

Comment:
Fix typo please

`[![SiameseCat]`

Reply:
Fixed ))) Is it ok, or I don't see something else wrong? )

Reply:
fixed. nice job)

Reply:
Thank you!